src/helpers/github.ts

```diff
@@ -71,10 +74,22 @@ export const createIssue = async (
     return { data, assignees: assignees !== null && assignees.length > 0 };
   } catch (error) {
     console.log("Error creating issue:", error);
-    return { data: null, assignees: false };
+    return { data: null, assignees: false, error };
   }
 };
 
+export const createGithubTelegramLink = async (username: string, telegramId: number, group: number, origin: string) => {
+  const id = generateRandomId(20);
```

Comment:
How random is this random?

Reply:
> How random is this random?

Just random enough to not be generated to frontrun any user, its 20 alphanumeric character that masks the telegram user and the data behind the scene

Reply:
> > How random is this random?
> 
> Just random enough to not be generated to frontrun any user, its 20 alphanumeric character that masks the telegram user and the data behind the scene

These are ephemeral right? There wont be some type of downstream issue if the same sequence is generated in a future conversation?

Reply:
> These are ephemeral right? There wont be some type of downstream issue if the same sequence is generated in a future conversation with a different person?

No, there won;t be any problem.. besides its a temporary id set in a session and deleted afterwards

Reply:
`crypto` is included with the newer nodejs. Doesn't need a separate package install.  Can we use it? 
```typescript
import crypto from 'crypto';
const id = crypto.randomUUID();
```

Reply:
> `crypto` is included with the newer nodejs. Doesn't need a separate package install. Can we use it?

There's no support for crypto on cloudflare workers yet

Reply:
I'm disappointed with how many limitations cloudflare workers seems to have. 

I feel like every day I learn something new that it does not support.

Reply:
> There's no support for crypto on cloudflare workers yet

https://developers.cloudflare.com/workers/runtime-apis/web-crypto/

Reply:
> > There's no support for crypto on cloudflare workers yet
> 
> https://developers.cloudflare.com/workers/runtime-apis/web-crypto/

Thank you, that worked..

I am going to switch back to `VARCHAR(255)` for key table as there was no way to limit the number of random string

Reply:
I think uuid length is 36 characters but need some testing before being sure https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID#return_value